sklearn/cluster/_spectral.py

```diff
@@ -599,6 +599,9 @@ class SpectralClustering(ClusterMixin, BaseEstimator):
     >>> clustering
     SpectralClustering(assign_labels='discretize', n_clusters=2,
         random_state=0)
+
+    For a more extended example, see:
```

Comment:
This example demonstrates the use of the function `spectral_clustering`, specifically the differences between the "assign_labels" options. Therefore a link to this example is best suited [here](https://github.com/scikit-learn/scikit-learn/blob/ddc170d41/sklearn/cluster/_spectral.py#L283) with a short description of what this example tries to show.